README.md

```diff
@@ -209,28 +199,22 @@ Feel free to check the [issues page](https://github.com/Tekhlay/react-bookstore/
 
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
-
-If you like this project...
+If you like this project please give a⭐️ and share with your friends.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank @Microverse
+I would like to thank @Microverse.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- FAQ (optional) -->
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/214248393-2b67c1f5-aa91-4eeb-9ca3-ee9e1d6875c4.png)

**[Optional]**

- [ ] Kindly remove this note shown in the image above from your readme file to make your readme file look clean :+1: